service/node.go

```diff
@@ -973,23 +974,27 @@ func (s *service) verifyAndUpdateInitiatorsInADiffHost(ctx context.Context, symI
 						log.Infof("UpdateHostName processing: %s to %s", initiator.HostID, hostID)
 						_, err := pmaxClient.UpdateHostName(ctx, symID, initiator.HostID, hostID)
```

Comment:
Our code would make sure that all the initiators are part of the same host. And if _modifyHostName_ is set to true, it will modify just one host.
With your changes, this scenario is more ambiguous where we can end up modifying hostnames even when not needed and it may become a random choice to update the host.

Reply:
Hi,
Looking at the code this will use unused initiators, so this looks good from the driver's perspective.
We will run our qualifications against this code and come back with the observations.